test/extended/operators/check_crd_status.go

```diff
@@ -0,0 +1,115 @@
+package operators
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	v1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	apiextensionsclientset "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// check_crds returns a list of names of CRDs that don't have subresource.status
+// For now, it ignores the ones that are currently failing.
+func check_crds(crd_item_list []v1.CustomResourceDefinition) []string {
+
+	contains := func(str_list []string, name string) bool {
+		for _, item := range str_list {
+			if name == item {
+				return true
+			}
+		}
+		return false
+	}
+
+	// These 17 CRDs do not have subresource.status.
+	//
```

Comment:
I would elaborate here that this was at the time the test was written, we want to prevent any more from getting in.

Reply:
and these should be tidied up over time.